models/Group.ts

```diff
@@ -262,13 +299,11 @@ export default function (sequelize, DataTypes): GroupStatic {
     group,
     stationsToAdd,
     applyToRecordingsFromDate
-  ): Promise<StationId[]> {
-    if (!(await group.userPermissions(authUser)).canAddStations) {
-      throw new AuthorizationError(
```

Comment:
Are you still checking user authorization here?  Sorry a bit hard to tell.

Reply:
Yep, just moved it up to the api middleware layer, since it seems more appropriate there.